micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalFxConfig.java

```diff
@@ -64,14 +64,25 @@ default String accessToken() {
     }
 
     /**
-     * @return {@code true} if the SignalFx registry should emit cumulative histogram
-     * buckets.
+     * If both "publishCumulativeHistogram" and "publishDeltaHistogram" are set, then delta will be used.
```

Comment:
The both thing is confusing...but I can live with it. Preference would be to have a single setting (something like `histogramBucketingStrategy`) with an enumeration of values like `delta` and `cumulative`.

Reply:
Correct, but the previous one already exists, so will be a breaking change.